lib/experimental/fonts/font-library/class-wp-rest-autosave-fonts-controller.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * WP_REST_Autosave_Fonts_Controller class.
+ *
+ * This file contains the class for the REST API Autosave Font Families and Font Faces Controller.
+ *
+ * @package    WordPress
+ * @subpackage REST_API
+ * @since      6.5.0
+ */
+
+if ( class_exists( 'WP_REST_Autosave_Fonts_Controller' ) ) {
+	return;
+}
+
+/**
+ * Autosave Font Families Controller class.
+ *
+ * @since 6.5.0
+ */
+class WP_REST_Autosave_Fonts_Controller {
+	public function register_routes() {
+		// disable autosave endpoints for font families and faces.
+	}
+}
```

Comment:
Here I'm making the empty autosave controller generic so we can use it for both `wp_font_family` and `wp_font_face` post types.